src/components/request/RequestForm.tsx

```diff
@@ -46,23 +92,40 @@ const RequestForm = () => {
           <ParametersValuesWrapper>
             <ParametersValueHeader>Proposal bond:</ParametersValueHeader>
             <ParametersValue>
-              <BondLogo src={usdcLogo} alt="bond_img" /> USDC 10000
+              <BondLogo src={usdcLogo} alt="bond_img" /> FAKE{" "}
+              {requestState.bond
+                ? ethers.utils.formatUnits(requestState.bond, 18)
+                : ""}
             </ParametersValue>
           </ParametersValuesWrapper>
           <ParametersValuesWrapper>
             <ParametersValueHeader>Proposal reward:</ParametersValueHeader>
             <ParametersValue>
-              <BondLogo src={usdcLogo} alt="bond_img" /> USDC 1000
+              <BondLogo src={usdcLogo} alt="bond_img" /> FAKE
+              {requestState.reward
+                ? ethers.utils.formatUnits(requestState.reward, 18)
+                : ""}
             </ParametersValue>
           </ParametersValuesWrapper>
           <ParametersValuesWrapper>
             <ParametersValueHeader>Liveness period:</ParametersValueHeader>
-            <ParametersValue>48 hours</ParametersValue>
+            <ParametersValue>
+              {requestState.customLiveness &&
```

Comment:
I think when this doesnt exist, it goes to oracle default liveness, but that might be something the sdk can populate

Reply:
I think all that matters is comparing the expiration time to the current time, no?